tests/Rules.Framework.Tests/Evaluation/Compiled/CompilationRulesSourceMiddlewareTests.cs

```diff
@@ -303,10 +303,7 @@ public async Task HandleGetRulesAsync_GivenArgsFilteringToRulesWithUncompiledCon
                 .WhoseValue.Should().BeOfType<Func<EvaluationContext<ConditionType>, bool>>();
             nextDelegateWasInvoked.Should().BeTrue();
 
-            Mock.Get(ruleConditionsExpressionBuilder)
-                .Verify(x => x.BuildExpression(It.IsAny<IConditionNode<ConditionType>>()), Times.Once());
-            Mock.Get(rulesDataSource)
-                .Verify(x => x.UpdateRuleAsync(It.IsAny<Rule<ContentType, ConditionType>>()), Times.Once());
+            Mock.VerifyAll();
```

Comment:
I think in order for the mocks to be validated we have to pass the list of mocks:
![image](https://user-images.githubusercontent.com/23389613/230035500-70f986ad-b0fc-44b0-aac5-4498f9629b55.png)

In this case meaning we have to write:
 Mock.VerifyAll(ruleConditionsExpressionBuilder, rulesDataSource);

Reply:
Please check the others that were modified.

Reply:
Understood, thanks for this note, I will fix that 👌

Reply:
Fixed ✅